auth/clientcredentials/connectrpc.go

```diff
@@ -170,9 +170,6 @@ func extractSchemaRequiredScopes(spec connect.Spec, extension *protoimpl.Extensi
 		return nil, errors.Newf("extension field %s not valid", extension.TypeDescriptor().FullName())
 	}
 	list := value.List()
-	if list.Len() == 0 {
-		return nil, errors.Newf("extension field %s cannot be empty", extension.TypeDescriptor().FullName())
-	}
```

Comment:
@unknwon FYI I had to add this to allow us to have RPCs that don't automatically check for a specific scope in the middleware; see the updated tests